nutkit/frontend/driver.py

```diff
@@ -128,16 +131,20 @@ def close(self):
 
     def session(self, access_mode, bookmarks=None, database=None,
                 fetch_size=None, impersonated_user=None,
-                ignore_bookmark_manager=None):
+                bookmark_manager: Optional[BookmarkManager] = None):
```

Comment:
```suggestion
                bookmark_manager=None):
```
I'm against type hinting only half of method signature and we still don't have a checker in place that could've flagged this or any mistake, really.

Reply:
My auto-complete is sad now, but I will accept the suggestion.

Reply:
It was the wrong type anyway :grimacing: It was a protocol BMM but should've been a frontend BMM...